src/components/blocks/ovc/dvm-application-offers.js

```diff
@@ -0,0 +1,75 @@
+import React from "react"
```

Comment:
It looks like all four components could be accomplished with one component. Recommend having one block component called dvm-application.js, then pass the colourOptions and icon (i.e., fa-circle-1) as variables in yamlWidget.js.